torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6553,6 +6553,41 @@ TEST_F(NVFuserTest, FusionExpandedInputThrow_CUDA) {
           "The expanded dim and the dim before it can not be contiguous.")));
 }
 
+// Repro for
+// https://github.com/csarofeen/pytorch/issues/1843#issuecomment-1270759724
```

Comment:
This test is pulled from https://github.com/csarofeen/pytorch/pull/2044, it is "fixed" by my previous PR https://github.com/csarofeen/pytorch/pull/2049 by not vectorizing 🤣 🤣 🤣 🤣 🤣